packages/helm-charts/oracle/templates/statefulset.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "name" . }}
+  labels:
+{{ include "labels" . | indent 4 }}
+{{ include "oracle-client-component-label" . | indent 4 }}
+spec:
+  clusterIP: None
+  selector:
+{{ include "oracle-client-component-label" . | indent 4 }}
+---
+apiVersion: apps/v1
+kind: StatefulSet
```

Comment:
Is this currently being used anywhere or is this for the future?

Reply:
It is currently being used-- it's a statefulset for creating oracle pods 0 through n-1